Source/debug.cpp

```diff
@@ -298,6 +307,7 @@ std::vector<DebugCmdItem> DebugCmdList = {
 	{ "refill", "Refills health and mana.", "", &DebugCmdRefillHealthMana },
 	{ "dropunique", "Attempts to generate unique item {name}.", "{name}", &DebugCmdGenerateUniqueItem },
 	{ "dropitem", "Attempts to generate item {name}.", "{name}", &DebugCmdGenerateItem },
+	{ "visit", "Interacts with a NPC whose name contains {name}.", "{name}", &DebugCmdTalkToTowner},

```

Comment:
woudn't talkto make more sens as a name for this command?

Reply:
aye